spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc

```diff
@@ -306,70 +301,59 @@ mysql> select * from test.messages;
+
+For situations where the data is consumed and processed between two different message brokers, Spring Cloud Data Flow provides easy-to-override global configurations, an out-of-the-box link:https://github.com/spring-cloud-stream-app-starters/bridge[`bridge-processor`], and DSL primitives to build these type of topologies.
 
-Let's assume we have data queueing up in RabbitMQ _(e.g., queue = `fooRabbit`)_ and the requirement
-is to consume all the payloads and publish them to Apache Kafka _(e.g., topic = `barKafka`)_, as the
-destination for downstream processing.
+Assume that data is queueing up in RabbitMQ (for example, queue = `myRabbit`) and the requirement is to consume all the payloads and publish them to Apache Kafka (for example, topic = `myKafka`), as the destination for downstream processing.
 
-Follow the global application of <<streams.adoc#spring-cloud-dataflow-global-properties, configurations>>
-to define multiple binder configurations.
+In that case, you should follow the global application of <<streams.adoc#spring-cloud-dataflow-global-properties, configurations>> to define multiple binder configurations, as shown i the following configuration
 
 [source,properties]
 ----
-# Apache Kafka Global Configurations (i.e., identified by "kafka1")
+# Apache Kafka Global Configurations (that is, identified by "kafka1")
 spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.kafka1.type=kafka
 spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.kafka1.environment.spring.cloud.stream.kafka.binder.brokers=localhost:9092
 spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.kafka1.environment.spring.cloud.stream.kafka.binder.zkNodes=localhost:2181
 
-# RabbitMQ Global Configurations (i.e., identified by "rabbit1")
+# RabbitMQ Global Configurations (that is, identified by "rabbit1")
 spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.rabbit1.type=rabbit
 spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.rabbit1.environment.spring.rabbitmq.host=localhost
 spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.rabbit1.environment.spring.rabbitmq.port=5672
 ----
 
-NOTE: In this example, both the message brokers are running locally and reachable at `localhost`
-with respective ports.
+NOTE: In the preceding example, both message brokers are running locally and are reachable at `localhost` through their respective ports.
 
-These properties can be supplied in a ".properties" file that is accessible to the server directly or via
-`config-server`.
+These properties can be supplied in a `.properties` file that is accessible to the server directly or through `config-server`, as follows:
 
 [source,bash]
 ----
 java -jar spring-cloud-dataflow-server-local/target/spring-cloud-dataflow-server-local-1.1.4.RELEASE.jar --spring.config.location=<PATH-TO-FILE>/foo.properties
```

Comment:
We could use the same trick to list the dynamic version as opposed to a very old `1.1.4.RELEASE` reference. The `wget` commands in the ref. guide include the trick.

Reply:
done